### PR TITLE
Improve naming

### PR DESCRIPTION
**Pull request by Ndpnt**
- Make code easier to understand by clarifying boundaries between `archivist` module and `open-terms-archive` app.
- Avoid using domain terms in logs

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -13,13 +13,13 @@ const modifiedOnly = args.includes('--modified-only');
 const refilterOnly = args.includes('--refilter-only');
 const schedule = args.includes('--schedule');
 
-(async () => {
-  const app = new Archivist();
+(async function OpenTermArchive() {
```
Comment (MattiSG) [inline src/index.js:16]:
```suggestion
(async function startOpenTermsArchive() {
```

**Review by MattiSG — CHANGES_REQUESTED** [review]
Typo in function name 😅

**Review by MattiSG — APPROVED** [review]
👍

**Review by martinratinaud — APPROVED** [review]
Test on local.

**Pull request merged**